### PR TITLE
add improve data and discuss to issue templates

### LINKED ISSUE #233: Add "improve data", "discuss", etc to create issue templates
### Description

I noticed that OpenRefine has options during issue creation to go to external links for chat, etc. We should do the same?

 - report a data error?
 - discuss (on Wikidata?)
 - Improve data
 - ...

Screenshot 2022-09-16 at 14-28-45 New Issue · OpenRefine_OpenRefine

### PR DESCRIPTION
**Pull request by dejmedus**
Hi there, I have added a config.yml with external links for "report a data error", "discuss the website", and "discuss data modeling".

Should close #233

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Report a data error
+    url: https://www.wikidata.org/w/index.php?action=edit&title=Wikidata:WikiProject_Govdirectory/Error_reports&section=new&preloadtitle=Rep%20ort+of+data+error&preload=Wikidata%3AWikiProject_Govdirectory%2FError_reports%2FError_report_template
```
Comment (Abbe98) [inline .github/ISSUE_TEMPLATE/config.yml:4]: I would suggest changing "preloadtitle=Rep%20ort+of+data+error" to "preloadtitle=Report+a+data+error".

**Review by Abbe98 — CHANGES_REQUESTED** [review]
Thank you @dejmedus! I only have a minor suggestion!

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (dejmedus) [reply .github/ISSUE_TEMPLATE/config.yml]: preloadtitle has been changed

**Review by Ainali — APPROVED** [review]

**Review by Abbe98 — APPROVED** [review]
Thank you @dejmedus :star:

**Pull request merged**